tests/test_task.py

```diff
@@ -88,44 +89,37 @@ def test_param_getter_parses_single_param_val_on_base_model_payload(mock_model,
 def test_param_script_portion_adds_formatted_json_calls(op):
     t = Task('t', op, [{'a': 1}])
     script = t.get_param_script_portion()
-    assert script == 'import json\na = json.loads(\'{{inputs.parameters.a}}\')\n'
+    assert script == f"import json\n{create_param_extraction('a')}"
```

Comment:
I love the replacements! 🚀 What do you think about including some small tests for `create_param_extraction` independently?

Reply:
woo! i could definitely do that. is there an edge case the current tests in `test_task.py` don't cover?